src/__tests__/index.js

```diff
@@ -251,4 +251,33 @@ import LeftPad from 'left-pad-js'
   assert.match(container.innerHTML, 'this is left pad')
 })
 
+test('require from current directory', async () => {
+  const mdxSource = `
+# Title
+
+import {Sample} from './other/sample-component'
+
+<Sample />
+`.trim()
+
+  const {code} = await bundleMDX(mdxSource, {
+    cwd: process.cwd(),
+    esbuildOptions: options => {
+      options.loader = {
+        ...options.loader,
+        '.png': 'dataurl',
```

Comment:
Maybe we should update our logic to default to `dataurl` for extensions we don't recognize so folks don't need to do this?

Reply:
I'm really not sure on this one.

If someone un-intentially supported an image it would be great for mdx-bundler to automatically handle that. On the other hand getting images to work in your MDX is going to require some extra config anyway so adding file types would be a small part of that.